CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ and this project adheres to
   - [#4614](https://github.com/bpftrace/bpftrace/pull/4614)
 - ast: Avoid undefined behavior in literal folding
   - [#4649](https://github.com/bpftrace/bpftrace/pull/4649)
+- Fix missing map key buffers for map aggregate functions
+  - [#4670](https://github.com/bpftrace/bpftrace/pull/4670)
+- Fix reference binding to misaligned address
```

Comment:
IIUC, this doesn't have any user-facing impact (at least not one that we know of). I'd drop this entry.